dev-lang/fpc/fpc_bin-2.6.2.recipe

```diff
@@ -38,9 +67,13 @@ SOURCE_DIR="fpc-2.6.2.i386-haiku"
 BUILD()
 {
 	echo "Please put as prefix: $prefix"
+	#give user time to copy prefix
+	sleep 5s
```

Comment:
This was there before, I have no idea how to fix it. But I'll try to make it work.